tools/README.md

```diff
@@ -63,7 +63,28 @@ node ./bin/3d-tiles-tools.js glbToB3dm -i ./specs/data/CesiumTexturedBox/CesiumT
 | ---- | ----------- | -------- |
 |`-i`, `--input`| Input path of the `.glb`| :white_check_mark: Yes |
 |`-o`, `--output`| Output path of the resulting `.b3dm` | No |
-|`-f`, `--force`|Overwrite output file if it exists.|No, default `false`|
+|`-f`, `--force`|Overwrite output file if it exists.| No, default `false` |
+
+### optimizeB3dm
+
+Optimize a b3dm using [gltf-pipeline](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/README.md). Since this tool does not
+process an entire tileset, it cannot be used with the Pipeline tool.
```

Comment:
@lilleyse can you add to the roadmap a traversal utility that can be used to apply per-tile stages like this to all tiles in a tileset?